ClassevivaPCTO/Dialogs/NoticeDialogContent.xaml.cs

```diff
@@ -117,7 +118,7 @@ private InfoBarSeverity SignAlertSeverityStatus
                 {
                     if (CurrentReadResult.reply.replSign.GetValueOrDefault())
                         return InfoBarSeverity.Success;
-                    return InfoBarSeverity.Error;
+                    return InfoBarSeverity.Informational;
```

Comment:
con questa modifica il caso di rifiuto non viene gestito in alcun modo

Reply:
aspetta che non avevo pushato altre modifiche